src/drivers/px4io/px4io.cpp

```diff
@@ -1213,59 +962,104 @@ void PX4IO::update_params()
 
 	char str[17];
 
-
-	// PWM_MAIN_MINx
-	{
-		pwm_output_values pwm{};
-		pwm.channel_count = _max_actuators;
-
-		for (unsigned i = 0; i < _max_actuators; i++) {
+	for (unsigned i = 0; i < _max_actuators; i++) {
+		// PWM_MAIN_MINx
+		{
 			sprintf(str, "%s_MIN%u", prefix, i + 1);
```

Comment:
I have this implemented in MixingOutput now; I guess that refactor can wait for my larger PR to be ready